website/docs/getting-started/installation.md

```diff
@@ -86,6 +86,6 @@ public void Configure(
 }
 ```
 
-Now you can create Message Handlers or access Producers and start exchanging events trouh Kafka.
+Now you can create Message Handlers or access Producers and start exchanging events trough Kafka.
```

Comment:
I believe there's just a small typo here 🙂 

```suggestion
Now you can create Message Handlers or access Producers and start exchanging events through Kafka.
```

Reply:
You're right. Fixed, please check.